utils/rpms/daos.spec

```diff
@@ -364,6 +365,9 @@ getent passwd daos_server >/dev/null || useradd -M daos_server
 %{_libdir}/*.a
 
 %changelog
+* Tue Jun 16 2020 Jonathan Martinez Montes <jonathan.martinez.montes@intel.com> - 1.1.0-24
+- Add new daos utility binary
```

Comment:
I think calling it by its name would be better since we already have a daos utility binary that is used to create containers (named 'daos').

Reply:
Done